aries-site/src/components/headings/DescriptiveHeader.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Heading, Paragraph, Box } from 'grommet';
+
+export const DescriptiveHeader = ({ subText, icon, title, ...rest }) => {
+  return (
+    <Box fill direction="row" gap="large" pad={{ vertical: 'large' }} {...rest}>
```

Comment:
```suggestion
    <Box fill direction={size !== 'small' ? 'row' : 'column'} gap="large" pad={{ vertical: 'large' }} {...rest}>
```
For mobile, the Icon should stack on top of the text.